test/node/parsing/WebGL/6

```diff
@@ -0,0 +1,552 @@
+typedef ([AllowShared] Uint32Array or sequence<GLuint>) Uint32List;
```

Comment:
This line is currently causing problems in parsing. The [W3C Grammar](https://www.w3.org/TR/WebIDL-1/#idl-grammar) does seem to have a rule for `[ExtendedAttributes]` proceeding a type in a `typedef` definition.